src/release/releaseHelper.d

```diff
@@ -169,9 +169,9 @@ string buildReleaseNotes ( string previous_version, string previous_notes  )
 
     import std.stdio;
 
-    immutable MigrationHeader = "Migration Instructions\n----------------------\n\n";
-    immutable DeprecationsHeader = "Deprecations\n------------\n\n";
-    immutable FeaturesHeader = "Features\n--------\n\n";
+    immutable MigrationHeader = "# Migration Instructions\n";
+    immutable DeprecationsHeader = "# Deprecations\n";
+    immutable FeaturesHeader = "# Features\n";
```

Comment:
Why? For me this is less readable in e-mails...

Reply:
It was incorrect, using h2 for both individual note headers and section headers. It was quite visible in rendered release notes at GitHub.

Maybe you should configure your mail client to render markdown emails? :trollface:

Reply:
> It was incorrect, using h2 for both individual note headers and section headers. It was quite visible in rendered release notes at GitHub.

Oh, I didn't know Markdown defined `-` as h2 and `=` as h1 (I come from reST where the meaning is contextual, the first one that appears in the document is h1, the second one h2, etc.). So can you replace it with `====` instead?